<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasureOneClass</class>
 <widget class="QMainWindow" name="MeasureOneClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MeasureOne</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>733</width>
         <height>682</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="ShowWidget" name="widgShow" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actCapture"/>
   <addaction name="actConnectMotion"/>
   <addaction name="separator"/>
   <addaction name="actRecord"/>
   <addaction name="actSnipping"/>
   <addaction name="separator"/>
   <addaction name="actTest"/>
   <addaction name="actTry"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockMotion">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QToolButton" name="btnMotionForw">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="measureone.qrc">
             <normaloff>:/MeasureOne/Resources/up_arrow_237px_1205504_easyicon.net.png</normaloff>:/MeasureOne/Resources/up_arrow_237px_1205504_easyicon.net.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="btnMotionLeft">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="measureone.qrc">
             <normaloff>:/MeasureOne/Resources/left_arrow_136px_1205449_easyicon.net.png</normaloff>:/MeasureOne/Resources/left_arrow_136px_1205449_easyicon.net.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="toolButton_9">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="measureone.qrc">
             <normaloff>:/MeasureOne/Resources/player_stop_128px_33504_easyicon.net.png</normaloff>:/MeasureOne/Resources/player_stop_128px_33504_easyicon.net.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="btnMotionRight">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="measureone.qrc">
             <normaloff>:/MeasureOne/Resources/right_arrow_140px_1205481_easyicon.net.png</normaloff>:/MeasureOne/Resources/right_arrow_140px_1205481_easyicon.net.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QToolButton" name="btnMotionBack">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="measureone.qrc">
             <normaloff>:/MeasureOne/Resources/down_arrow_228px_1205407_easyicon.net.png</normaloff>:/MeasureOne/Resources/down_arrow_228px_1205407_easyicon.net.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QToolButton" name="btnMotionUp">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="measureone.qrc">
             <normaloff>:/MeasureOne/Resources/upload_162px_1205510_easyicon.net.png</normaloff>:/MeasureOne/Resources/upload_162px_1205510_easyicon.net.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnMotionDown">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="measureone.qrc">
             <normaloff>:/MeasureOne/Resources/download_158px_1205413_easyicon.net.png</normaloff>:/MeasureOne/Resources/download_158px_1205413_easyicon.net.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QCheckBox" name="chkMixSrc">
         <property name="text">
          <string>叠加原图</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btnPreGauss">
         <property name="text">
          <string>高斯滤波</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPreMiddle">
         <property name="text">
          <string>中值滤波</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="btnAlgorOtsu">
         <property name="text">
          <string>OTSU</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAlgorThres">
         <property name="text">
          <string>自适应阈值法</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actCapture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="measureone.qrc">
     <normaloff>:/MeasureOne/Resources/Play_128px_1183387_easyicon.net.png</normaloff>:/MeasureOne/Resources/Play_128px_1183387_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>开始采集</string>
   </property>
  </action>
  <action name="actConnectMotion">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="measureone.qrc">
     <normaloff>:/MeasureOne/Resources/Link_154.17744360902px_1183375_easyicon.net.png</normaloff>:/MeasureOne/Resources/Link_154.17744360902px_1183375_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>开始移动</string>
   </property>
  </action>
  <action name="actRecord">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="measureone.qrc">
     <normaloff>:/MeasureOne/Resources/Cinema_Camera_111.54057428215px_1183349_easyicon.net.png</normaloff>:/MeasureOne/Resources/Cinema_Camera_111.54057428215px_1183349_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>录像</string>
   </property>
  </action>
  <action name="actSnipping">
   <property name="icon">
    <iconset resource="measureone.qrc">
     <normaloff>:/MeasureOne/Resources/Camera_146.49499284692px_1183360_easyicon.net.png</normaloff>:/MeasureOne/Resources/Camera_146.49499284692px_1183360_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>截图</string>
   </property>
  </action>
  <action name="actTest">
   <property name="icon">
    <iconset resource="measureone.qrc">
     <normaloff>:/MeasureOne/Resources/Question_127.84019975031px_1183370_easyicon.net.png</normaloff>:/MeasureOne/Resources/Question_127.84019975031px_1183370_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </action>
  <action name="actTry">
   <property name="icon">
    <iconset resource="measureone.qrc">
     <normaloff>:/MeasureOne/Resources/Settings_128.48120300752px_1183382_easyicon.net.png</normaloff>:/MeasureOne/Resources/Settings_128.48120300752px_1183382_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>Try</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ShowWidget</class>
   <extends>QWidget</extends>
   <header>showwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="measureone.qrc"/>
 </resources>
 <connections/>
</ui>
